### DISCUSSION TITLE
[desktop] ScrollView content visible outside the scroll area.

### DISCUSSION THREAD

**Discussion by mcNets** [unoplatform/uno#18289] [Q&A]
Working in my last app I noticed that sometimes the content of a ScrollView or part of the content, in this case just the Border, is visible outside the scroll area.

I'm using a FeedView where the NormalTemplate has a GridView inside a ScrollView.

**How to reproduce**

Download the attached solution and execute the desktop project, then scroll up the content by clicking the small arrow at the bottom of the scrollbar. The card's border is visible outside the scroll area.

In my application it happens scrolling the content using the scrollbar no matter where, in this one, at least in my computer only by clicking the bottom arrow.

![TestScrollView](https://github.com/user-attachments/assets/ae90dac5-2dfe-45e4-831f-da51291a9304)

[TestScrollView.zip](https://github.com/user-attachments/files/17121907/TestScrollView.zip)

**Comment by agneszitte** [comment]
@mcNets I am not able to reproduce the issue for Desktop while following the repro steps you shared while testing on Windows OS with the sample you provided (I am using Visual Studio 2022 Version 17.11.4 to build/launch the Desktop project here)
Maybe I have missed a detail, can you provide more info or a video to help me better understand how you can repro it please
Thanks a lot in advance

**Reply by mcNets** [reply]
@agneszitte  Let me try to make a video

**Reply by mcNets** [reply]
@agneszitte let me know if there is any problem. Thanks.

[test_scroll_view.zip](https://github.com/user-attachments/files/17134955/test_scroll_view.zip)

**Reply by agneszitte** [reply]
@mcNets Thank you a lot for the video. I gave it another try and still I am not able to reproduce on my side (see video below of my test).
@jeromelaban, @MartinZikmund what do you think could be different in this case on @mcNets side please?

https://github.com/user-attachments/assets/52392da4-a2bd-48b1-9e38-f5d58cb247c1

**Reply by mcNets** [reply]
@agneszitte I'm using:

- VS Version 17.12.0 Preview 2.0 
- VSCode Version: 1.94.0-insider (user setup) Commit: f35c3823e3b7ea4c52f7fee4659bcce39b42ce9e

Have you tried on Dark theme?

**Reply by agneszitte** [reply]
@mcNets Same with Dark Theme on my side. I will give a quick try with VS Version 17.12.0 Preview 2.0 on my side just in case 
(to see if it makes any difference)
(Also I am testing on Windows 11 Pro Version 23H2 - OS Build 22631.4169 for reference)

**Reply by mcNets** [reply]
@agneszitte I'm also on Win 11 Pro 23H2

Let me add a Light theme to the control.

**Reply by mcNets** [reply]
@agneszitte I've added dark and light themes and this is the result:

The last project:
[TestScrollView.zip](https://github.com/user-attachments/files/17135620/TestScrollView.zip)

https://github.com/user-attachments/assets/d3209241-057c-4fc4-8049-2f720497054e

**Reply by mcNets** [reply]
```xml
{
  "msbuild-sdks": {
    "Uno.Sdk": "5.3.108"
  },
  "sdk":{
    "allowPrerelease": false
  }
}
```
Curiously only the Border appears out of the scroll area. 🤷‍♂️

**Reply by agneszitte** [reply]
@mcNets I am really not sure why this is happening on your side. I am still not able to repro it on mine (see attached video below).
I will see if anyone else from the Uno Platform Team can reproduce it.

Recap of details for what I tested with so far for the Desktop platform:
- Light/Dark Theme
- Uno.Sdk version 5.3.108 (I tested your sample apps as is without any additional changes)
- Visual Studio 2022 Version 17.11.4 & Version 17.12.0 Preview 2.0
- On Windows 11 Pro Version 23H2 - OS Build 22631.4169

https://github.com/user-attachments/assets/bf458f83-eaef-4602-9f1f-02d55fc0d999

**Reply by agneszitte** [reply]
@mcNets I have opened an issue from this discussion so we can investigate as @Xiaoy312  is able to reproduce it using mouse wheel scroll. Details will be tracked with: https://github.com/unoplatform/uno/issues/18299
